app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,24 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/8f76becb1e.js" crossorigin="anonymous"></script>
```

Comment:
- [ ] It could be my machine but there is an error mkong the app not run. Kindly fix the syntax error currently causing the app to not run
![image](https://user-images.githubusercontent.com/81932478/211173266-63c00d61-309a-4156-8450-6015421fc509.png)

Reply:
Kindly install tailwind at this link https://tailwindcss.com/docs/guides/ruby-on-rails